apps/web/prisma/schema/workspace.prisma

```diff
@@ -56,14 +54,15 @@ enum Role {
 }
 
 model ProjectUsers {
-  id        String   @id @default(cuid())
-  role      Role     @default(member)
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  user      User     @relation(fields: [userId], references: [id], onDelete: Cascade)
-  userId    String
-  project   Project  @relation(fields: [projectId], references: [id], onDelete: Cascade)
-  projectId String
+  id                     String                  @id @default(cuid())
+  role                   Role                    @default(member)
+  createdAt              DateTime                @default(now())
+  updatedAt              DateTime                @updatedAt
+  user                   User                    @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId                 String
+  project                Project                 @relation(fields: [projectId], references: [id], onDelete: Cascade)
+  projectId              String
+  notificationPreference NotificationPreference?
```

Comment:
Btw does this mean that we should be doing DefaultDomains? on the Project table as well?